### ISSUE TITLE
[bug]: TypeError: unsupported operand type(s) for |: 'type' and 'type'

### ISSUE THREAD

**Issue by pdoane** [invoke-ai/InvokeAI#2796]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### What happened?

Running the 2.3.1 installer on a fresh directory fails in the Configuring InvokeAI step while Loading Python libraries...

- Moved old invokeai folder to invokeai_230
- Ran install.sh from InvokeAI-Installer
- Script fails with message `TypeError: unsupported operand type(s) for |: 'type' and 'type'`

### Screenshots

<img width="708" alt="image" src="https://user-images.githubusercontent.com/1621950/221251476-1f783e03-426d-4316-9619-502696a9504a.png">

### Additional context

```
python3 --version
Python 3.9.6
```

```
pip3 --version
pip 21.2.4 from /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/site-packages/pip (python 3.9)
```

**Comment by moono** [comment]
same here, while using docker installation too.

**Comment by rodrigoalviani** [comment]
+1
MacOS (M1) 13.1 version
Python 3.9.6
pip 23.0.1

**Comment by rodrigoalviani** [comment]
New posted release resolved this issue (2.3.1.post2)

**Comment by jameswan** [comment]
I am using InvokeAI 2.3.4.post1

but I still get the error:

  File "C:\Users\James\anaconda3\envs\invoke\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\James\anaconda3\envs\invoke\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\James\invokeai\.venv\Scripts\invokeai-batch.exe\__main__.py", line 4, in <module>
  File "C:\Users\James\invokeai\.venv\lib\site-packages\ldm\invoke\dynamic_prompts.py", line 329, in <module>
    stanza: str | dict | listconfig.ListConfig | dictconfig.DictConfig,
TypeError: unsupported operand type(s) for |: 'type' and 'type'
